### PR TITLE
fixed spelling errors on dumping SCR configuration parameters

### PR DESCRIPTION
**Pull request by hbchen1984**

### CODE REVIEW

File: src/scr.c [hunk 1]
```diff
@@ -678,18 +678,32 @@ static int scr_get_params()
   /* if not enabled, bail with an error */
   if (! scr_enabled) {
     scr_param_finalize();
+    if (scr_my_rank_world == 0) {
+      scr_dbg(1, "SCR_ENABLE=%d", scr_enabled);
+    }
```
Comment (adammoody) [inline src/scr.c]: Let's pull this out from the if block and move it up to line 677.

File: src/scr.c [hunk 2]
```diff
@@ -678,18 +678,32 @@ static int scr_get_params()
...
     return SCR_FAILURE;
   }
+  else {
+    if (scr_my_rank_world == 0) {
+      scr_dbg(1, "SCR_ENABLE=%d", scr_enabled);
+    }
+  }
```
Comment (adammoody) [inline src/scr.c]: Then we can delete this else branch to simplify
```suggestion
```

File: src/scr.c [hunk 3]
```diff
@@ -678,18 +678,32 @@ static int scr_get_params()
...
 
   /* set debug verbosity level */
   if ((value = scr_param_get("SCR_DEBUG")) != NULL) {
     scr_debug = atoi(value);
   }
+  if (scr_my_rank_world == 0) {
+     scr_dbg(1, "SCR_DEBUG=%d",scr_debug);
+  }
```
Comment (adammoody) [inline src/scr.c]: Just for a style thing, we typically use 2 spaces for indents in the code, but it looks like you might have 3 for all of the scr_dbg lines.  Would you please update all of these lines to use 2 spaces instead?
```suggestion
  if (scr_my_rank_world == 0) {
    scr_dbg(1, "SCR_DEBUG=%d", scr_debug);
  }
```

File: src/scr.c [hunk 4]
```diff
@@ -916,24 +993,33 @@ static int scr_get_params()
               __FILE__, __LINE__
       );
     }
-  }
+  }
```
Comment (adammoody) [inline src/scr.c]: Let's drop the trailing white space on this line
```suggestion
  }
```

File: src/scr.c [hunk 5]
```diff
@@ -1021,17 +1146,26 @@ static int scr_get_params()
...
     scr_flush_async = atoi(value);
   }
+  if (scr_my_rank_world == 0) {
+     scr_dbg(1, "SCR_FLUSH_ASYNC=%d", scr_flush_async);
+  }
 
-  /* Specify whether our flush will be finalized in poststage (currently
-   * only supported with BBAPI). */
+ /* Specify whether our flush will be finalized in poststage (currently
+    only supported with BBAPI). */
```
Comment (adammoody) [inline src/scr.c]: Just another code style thing: for multi-line comments, we align the stars
```suggestion
 /* Specify whether our flush will be finalized in poststage (currently
  * only supported with BBAPI). */
```

File: src/scr.c [hunk 6]
```diff
@@ -1122,6 +1286,19 @@ static int scr_get_params()
...
+
+  if (scr_my_rank_world == 0) {
+     kvtree_print_mode(scr_groupdesc_hash, 4, KVTREE_PRINT_KEYVAL);
+  }
+  if (scr_my_rank_world == 0) {
+     kvtree_print_mode(scr_storedesc_hash, 4, KVTREE_PRINT_KEYVAL);
+  }
+  if (scr_my_rank_world == 0) {
+     kvtree_print_mode(scr_reddesc_hash, 4, KVTREE_PRINT_KEYVAL);
+  }
```
Comment (adammoody) [inline src/scr.c]: Before each of the kvtree_print lines, let's add a note about what we're printing like so:
```suggestion
  if (scr_my_rank_world == 0) {
     scr_dbg(1, "Group descriptors:");
     kvtree_print_mode(scr_groupdesc_hash, 4, KVTREE_PRINT_KEYVAL);
  }
  if (scr_my_rank_world == 0) {
     scr_dbg(1, "Store descriptors:");
     kvtree_print_mode(scr_storedesc_hash, 4, KVTREE_PRINT_KEYVAL);
  }
  if (scr_my_rank_world == 0) {
     scr_dbg(1, "Redundancy descriptors:");
     kvtree_print_mode(scr_reddesc_hash, 4, KVTREE_PRINT_KEYVAL);
  }
```

File: src/scr.c [hunk 7]
```diff
@@ -1144,6 +1321,7 @@ static int scr_get_params()
   return SCR_SUCCESS;
 }
 
+
```
Comment (adammoody) [inline src/scr.c]: One more extra line to drop.
```suggestion
```

**Comment by adammoody** [comment]
@hbchen1984 , thanks!  I was able to test this, and it's looking good.  I took one more pass this time to recommend some code style items and a few other minor tweaks.  After you address those, I can help you squash this into one commit and it should be ready to merge.

File: src/scr.c [hunk 8]
```diff
@@ -767,6 +814,9 @@ static int scr_get_params()
   } else {
     scr_jobid = scr_env_jobid();
   }
+  if (scr_my_rank_world == 0) {
+     scr_dbg(1, "SCR_JOB_ID=%s", scr_jobid);
+  }
```
Comment (adammoody) [inline src/scr.c]: Let's move this down just below the following block to come at line 832.  That block can set scr_jobid if it is NULL.

File: src/scr.c (hunk 1 above)
Comment (hbchen1984) [reply src/scr.c]: new updated is submitted based on the comments.

**Review by adammoody — APPROVED** [review]

**Comment by adammoody** [comment]
Thanks @hbchen1984 !  Congrats on your first PR :-)

**Pull request merged**